src/spikeinterface/extractors/nwbextractors.py

```diff
@@ -884,12 +904,15 @@ def __init__(
         if units_ids is None:
             units_ids = units_table["id"].data
 
+        units_table = self._nwbfile.units
```

Comment:
```suggestion
```
This is already at line 877, right?

Reply:
Thanks,  I overused git magic and had duplication as a consequence of my sins. Will correct.

Reply:
Done, thanks